olp-cpp-sdk-core/src/http/ios/OLPNetworkIOS.mm

```diff
@@ -98,6 +98,7 @@
     case NSURLErrorDataNotAllowed:
       return ErrorCode::OFFLINE_ERROR;
     case NSURLErrorTimedOut:
+    case NSURLErrorNetworkConnectionLost:
```

Comment:
Is it not rather OFFLINE_ERROR?

Reply:
A client or server connection was severed in the middle of an in-progress load.

https://developer.apple.com/documentation/foundation/1508628-url_loading_system_error_codes/nsurlerrornetworkconnectionlost

Reply:
Yes, but this is not a time out, rather a connection lost. Maybe add a new error code to `http::ErrorCode` enum?

Reply:
@andescu, I checked other implementation, the error code should be IO_ERROR